Cargo.toml

```
@@ -23,11 +24,12 @@ cw721-base-016  = { git = "https://github.com/CosmWasm/cw-nfts", tag = "v0.16.0"
 cw721-metadata-onchain-016  = { git = "https://github.com/CosmWasm/cw-nfts", tag = "v0.16.0", package = "cw721-metadata-onchain" } # needed for testing legacy migration
 cw721-base-017  = { git = "https://github.com/CosmWasm/cw-nfts", tag = "v0.17.0", package = "cw721-base" } # needed for testing legacy migration
 cw721-base-018  = { git = "https://github.com/CosmWasm/cw-nfts", tag = "v0.18.0", package = "cw721-base" } # needed for testing legacy migration
-cw-multi-test   = { version = "^0.20", features = ["cosmwasm_1_2"] }
-cw-ownable      = { git = "https://github.com/public-awesome/cw-plus-plus.git", rev = "28c1a09bfc6b4f1942fefe3eb0b50faf9d3b1523"} # TODO: switch to official https://github.com/larry0x/cw-plus-plus once merged
-cw-paginate-storage = { version = "^2.4", git = "https://github.com/DA0-DA0/dao-contracts.git" }
-cw-storage-plus = "^1.1"
-cw-utils        = "^1.0"
+cw-multi-test   = { version = "^2.1", features = ["cosmwasm_2_0"] }
+# TODO: switch to official https://github.com/larry0x/cw-plus-plus
+# requires merge of https://github.com/larry0x/cw-plus-plus/pull/25
+cw-ownable      = { git = "https://github.com/mintthemoon/cw-plus-plus", branch = "dev", package = "cw-ownable" }
```

Noticed that mentioned PR got merged :eyes: 
```suggestion
cw-ownable      = { version = "^2.1" }
```